index.css

```diff
@@ -504,3 +563,346 @@ textarea {
+    width: 566px;
+    height: 24px;
+    margin: 24px 0 0 90px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .developer {
+    margin: 30px 0 0 0;
+  }
+
+  .theme1 {
+    color: #f4f5f7;
+    width: 440px;
+    height: 123px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .palet {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    list-style: none;
+  }
+
+  .palet a {
+    position: relative;
+    bottom: 350px;
+  }
+
+  .myresent {
+    display: flex;
+    margin-left: 120px;
+    background-image: url(../Desktop/line3.png);
+    background-repeat: no-repeat;
+    background-position: 80% 75%;
+    background-size: 63%;
+  }
+
+  footer h2 {
+    width: 450px;
+    height: 256px;
+    font-style: normal;
+    font-family: "Inter", sans-serif;
+    text-align: start;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    color: #172b4d;
+    margin: 0  0 0 150px;
+  }
+
+  .myresent h2 {
+    height: 52px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 110px;
+    background-image: none;
+  }
+
+  .box1 {
+    grid-column: 3;
+  }
+
+  .box1 img {
+    display: none;
+  }
+
+  .box1 .daily {
+    width: 410px;
+    height: 140px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 350;
+    font-size: 19px;
+    line-height: 26px;
+  }
+
+  .box2 {
+    height: 435px;
+    background: url(../Desktop/Placeholder.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box3 {
+    height: 435px;
+    background: url(../Desktop/Placeholder4.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box4 {
+    height: 435px;
+    background: url(../Desktop/Placeholder5.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box {
+    height: 435px;
+    background: url(../Desktop/Placeholder3.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .dashbord1 {
+    display: none;
+  }
+
+  .texto1 {
+    display: none;
+  }
+
+  .contenido {
+    display: grid;
+    grid-template-columns: repeat(3, 400px);
+    grid-gap: 10px;
+    background: url(Desktop/gymfit.png);
+    background-repeat: no-repeat;
+    background-position: 15% 0%;
+  }
+
+  .multi {
+    height: 48px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .brian1 button {
+    width: 369px;
+    height: 60px;
+    margin-top: 294px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    background-color: #ff6b00;
+    font-size: 17px;
+    color: #fff;
+  }
+
```

Comment:
- [ ] Kindly implement the button interactions to all your buttons.
![button2](https://user-images.githubusercontent.com/85095702/160023021-bb3cebae-e102-4a26-ab7d-1c08be19bd7b.PNG)

- [ ] Also for all the cards, please adjust the display of the "See Project" button for all cards in desktop view to match the Figma design. (You can make it so that the button only appears when the card is hovered over and the button has to be inside of the card)

Reply:
- [ ] As the last reviewer mentioned please add the correct styling for `:hover` and `:active` button interactions.